test/aws/rails/railtie_test.rb

```diff
@@ -3,29 +3,50 @@
 require 'test_helper'
 
 module Aws
+  # Test services namespaces
+  module Service1
+    Client = Aws::SES::Client.dup
+  end
+
+  module Service2
+    Client = Aws::SES::Client.dup
+  end
+
   module Rails
-    class RailtieTest < Minitest::Test
-      def test_add_action_mailer_delivery_method
-        assert_equal ::Aws::Rails::Mailer,
-                     ::ActionMailer::Base.delivery_methods[:ses]
+    describe 'Railtie' do
+      it 'adds action mailer delivery method' do
+        expect(ActionMailer::Base.delivery_methods[:ses]).to eq Aws::Rails::Mailer
       end
 
-      def test_log_to_rails_logger
-        assert_equal ::Rails.logger, Aws.config[:logger]
+      it 'sets the Aws logger' do
+        expect(Aws.config[:logger]).to eq ::Rails.logger
       end
 
-      def test_use_rails_encrypted_credentials
-        creds = ::Rails.application.credentials.aws
-        assert_equal Aws.config[:access_key_id], creds[:access_key_id]
-        assert_equal Aws.config[:secret_access_key], creds[:secret_access_key]
+      describe 'rails encrypted credentials' do
```

Comment:
should this instead be a context? if you are going for describe, use the method name.

Reply:
Context doesnt exist in the minitest/spec.